opstack/src/builder.rs

```diff
@@ -82,7 +82,10 @@ impl OpStackClientBuilder {
                 chain: ChainConfig {
```

Comment:
We should support building based on network similar to how we do it in Ethereum (see the ethereum builder for inspiration). So we should have a `network` method on the builder and pass in a `Network` enum to properly set everything from within the `ChainConfig`.